.travis.yml

```diff
@@ -28,7 +28,11 @@ jobs:
 
     - stage: storybook
       script:
-        - CHROMATIC_APP_CODE=$CHROMATIC_APP_CODE yarn run chromatic
+        # run chromatic only when a build does not originate from a pull request (push build), or when it is a pull request build originating from a fork
+        - |
+          if [[ $TRAVIS_PULL_REQUEST_SLUG != $TRAVIS_REPO_SLUG ]]; then
+            CHROMATIC_APP_CODE=$CHROMATIC_APP_CODE yarn run chromatic
```

Comment:
You shouldn't need to pass CHROMATIC_APP_CODE like that. In fact it will cause the code to be exposed in the Travis logs.

Reply:
Thanks @ghengeveld!